.changeset/lovely-bobcats-obey.md

```diff
@@ -0,0 +1,10 @@
+---
+"@justifi/webcomponents": minor
+---
+
+- Added new props to `Checkout` component:
+  - `disable-credit-card`: Disables credit card payment method
+  - `disable-bank-account`: Disables bank account payment method
+  - `disable-bnpl`: Disables BNPL payment method
+  - `disable-payment-method-group`: Disables saved payment methods
```

Comment:
How about adding option to the description of each prop:
Disables new credit card payment method option
Disables new bank account payment method option 
Disables BNPL payment method option
Disables saved payment method option